ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/mlag-configuration.j2

```diff
@@ -6,9 +6,12 @@ mlag configuration
    local-interface {{ mlag_configuration.local_interface }}
    peer-address {{ mlag_configuration.peer_address }}
 {%   if mlag_configuration.peer_address_heartbeat.peer_ip is defined and mlag_configuration.peer_address_heartbeat.peer_ip is not none %}
-   peer-address heartbeat {{ mlag_configuration.peer_address_heartbeat.peer_ip }}{% if mlag_configuration.peer_address_heartbeat.vrf is defined and mlag_configuration.peer_address_heartbeat.vrf is not none and mlag_configuration.peer_address_heartbeat.vrf is ne 'default' %} vrf {{ mlag_configuration.peer_address_heartbeat.vrf }}
-{%       endif -%}
-{%   endif %}
+{%   if mlag_configuration.peer_address_heartbeat.vrf is defined and mlag_configuration.peer_address_heartbeat.vrf is not none and mlag_configuration.peer_address_heartbeat.vrf is ne 'default' %}
+   peer-address heartbeat {{ mlag_configuration.peer_address_heartbeat.peer_ip }} vrf {{ mlag_configuration.peer_address_heartbeat.vrf }}
+{%    else %} {# using the default VRF #}
```

Comment:
Please move comment above the `{% else %}` statement, as opposed to in-line as per below. The space between the `{% else %}` statement and comments results in adding one extra whitespace on the command when rendered.
 
```
{# using the default VRF #}
{%    else %}
```

Reply:
Change completed